src/utils/yelpFusionApiUtils.ts

```diff
@@ -5,7 +5,7 @@ import { Image, ImageProviderName } from '../models/data_models/image.js';
 import { Geocode } from '../models/data_models/types.js';
 
 export class YelpFusionApiUtils {
-	protected yelpClient?: YelpClient;
+	protected yelpClient!: YelpClient;
```

Comment:
to avoid the `!` we could make the `YelpClient` an `init` parameter and create an async static `.from` method

Reply:
I guess this could apply to the Google utils as well